src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    abstract void doWork();

```

Comment:
Захар был не совсем прав, то что он написал относится к интерфейсу. В абстрактном классе нужно прописывать `public abstract`
```suggestion
   public abstract void doWork();

```